api/handlers.go

```diff
@@ -78,10 +88,15 @@ func (h *handler) getNodesByUnitIDHandler(w http.ResponseWriter, r *http.Request
 // /api/v1/system/health/units/:unit_id:/nodes/:node_id:
 func (h *handler) getNodeByUnitIDNodeIDHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
-	nodePerUnit, err := h.monitoringResponse.GetSpecificNodeForUnit(vars["unitid"], vars["nodeid"])
+	unitId := vars["unitid"]
+	nodePerUnit, err := h.monitoringResponse.GetSpecificNodeForUnit(unitId, vars["nodeid"])
```

Comment:
it's a little inconsistent that `unitId` is pulled out into a var but not `vars["nodeid"]` - I think maybe it was ok to leave it as originally written?

Reply:
Fixed